Updates mx.m3u

Carlinhos027 opened the pull request.

Carlinhos027 committed:
Update mx_televisa.m3u

Carlinhos027 committed:
Update mx.m3u

Added new channels & deleted Televisa Tijuana link because I moved it to mx_televisa.m3u

BellezaEmporium commented on streams/mx.m3u:
```diff
@@ -217,13 +215,23 @@ https://5e50264bd6766.streamlock.net/tabasco/videotabasco/playlist.m3u8
+https://ythls.onrender.com/channel/UCgu4v4HAioB0N5K6aq_H-WQ.m3u8
+#EXTINF:-1 tvg-id="XHFGLTDT.mx",El 15 TV (720p)
+https://stream.oursnetworktv.com/latin/encoder32/playlist.m3u8
+#EXTINF:-1 tvg-id="bitMe.mx",bitMe (720p)
```
```suggestion
#EXTINF:-1 tvg-id="bitMe.mx",bitMe (720p) [Geo-blocked]
```

BellezaEmporium commented on streams/mx.m3u:
```diff
@@ -217,13 +215,23 @@ https://5e50264bd6766.streamlock.net/tabasco/videotabasco/playlist.m3u8
+#EXTINF:-1 tvg-id="XHFGLTDT.mx",El 15 TV (720p)
+https://stream.oursnetworktv.com/latin/encoder32/playlist.m3u8
+#EXTINF:-1 tvg-id="bitMe.mx",bitMe (720p)
+https://live4-ott.izzigo.tv/out/u/hls/BIT-ME-HD/default.m3u8
```
Needs clearkey decryption.

Carlinhos027 committed:
Update mx_televisa 2

Added Televisa Guadalajara

Carlinhos027 committed:
Update mx 2

Added new channels, deleted bitMe link & removed Televisa Guadalajara and added alt link to N+ channels

BellezaEmporium committed:
Merge branch 'master' into Carlinhos027-patch-1

BellezaEmporium committed:
b version doesn't work for n+
